File: java/NodoFacultad.java
package java;
import java.util.Objects;

public class NodoFacultad {
    private Facultad facultad;
    private NodoFacultad sig,ant;


    public NodoFacultad() {
        this.sig = null;
        this.ant = null;
    }

    public Facultad getFacultad() {
        return this.facultad;
    }

    public void setFacultad(Facultad facultad) {
        this.facultad = facultad;
    }

    public NodoFacultad getSig() {
        return this.sig;
    }

    public void setSig(NodoFacultad sig) {
        this.sig = sig;
    }

    public NodoFacultad getAnt() {
        return this.ant;
    }

    public void setAnt(NodoFacultad ant) {
        this.ant = ant;
    }


    @Override
    public boolean equals(Object o) {
        if (o == this)
            return true;
        if (!(o instanceof NodoFacultad)) {
            return false;
        }
        NodoFacultad nodoFacultad = (NodoFacultad) o;
        return Objects.equals(facultad, nodoFacultad.facultad) && Objects.equals(sig, nodoFacultad.sig) && Objects.equals(ant, nodoFacultad.ant);
    }


}

File: c#/Interfaz Grafica/Guia_Centros_Estudio_Superior.Core/Models/NodoFacultad.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;


using System.IO;
namespace Guia_Centros_Estudio_Superior.Core.Models;
[Serializable]

public class NodoFacultad
{
    private Facultad facultad;
    private NodoFacultad sig,ant;

    public Facultad Facultad
    {
        get => facultad;
        set => facultad = value;
    }
    public NodoFacultad Sig
    {
        get => sig;
        set => sig = value;
    }
    public NodoFacultad Ant
    {
        get => ant;
        set => ant = value;
    }

    public NodoFacultad() {
        this.Sig = null;
        this.Ant = null;
    }

    public Facultad getFacultad() {
        return this.Facultad;
    }

    public void setFacultad(Facultad facultad) {
        this.Facultad = facultad;
    }

    public NodoFacultad getSig() {
        return this.Sig;
    }

    public void setSig(NodoFacultad sig) {
        this.Sig = sig;
    }

    public NodoFacultad getAnt() {
        return this.Ant;
    }

    public void setAnt(NodoFacultad ant) {
        this.Ant = ant;
    }





}
